Automattic/jetpack
# Jetpack DNA: Isolate Tracks Package

This PR moves more files into the Tracks Package.

**Testing Instructions**

* enable `WP_DEBUG_LOG` for your site
* navigate to the Jetpack Dashboard of your site
* make sure to perform actions that are tracked like activating modules
* make sure no errors or warnings are logged into the error_log
* try disconnecting and reconnecting Jetpack
* try a full sync
* try logging in with SSO
* do tests still pass?
* is your debug.log clean?
